### DISCUSSION TITLE
open Autocomplete Manually

### DISCUSSION THREAD

**Discussion by MR-Mostafa** [mantinedev/mantine#1609] [Q&A] [answered]
Is it possible to open manually the Autocomplete dropdown menu (programmatically)?
When I use `initiallyOpened` as a state, the dropdown does not appear when the state changes
How we can do this?

**Comment by rtivital** [comment, marked as answer]
No, Autocomplete and other component dropdowns cannot be controlled on your side, it is hard to implement this feature while keeping component accessible.

**Reply by MR-Mostafa** [reply]
Is there any way to handled this?
I absolutely need open drop-down programmatically.
Thank you for your help.

**Reply by rtivital** [reply]
You can get input ref and click on it programmatically: `inputRef.current.click()`

**Reply by MR-Mostafa** [reply]
Thank you for your assistance, the solution worked 🌷
